src/Strings/src/CalculatorStrings.cpp

```diff
@@ -1,25 +1,26 @@
 /**
  * @file CalculatorStrings.cpp
- * @author your name (you@domain.com)
- * @brief 
- * @version 0.1
- * @date 2021-09-13
- * 
- * @copyright Copyright (c) 2021
- * 
+ * @brief Contains CalculatorStrings constant strings
  */
 #include "CalculatorStrings.h"
 
+namespace calculator {
 
-namespace calculator
-{
-    constexpr char CalculatorStrings::ERROR_MESSAGE_DIVIDE_BY_ZERO[];
-    constexpr char CalculatorStrings::ERROR_MESSAGE_INVALID_INPUT[];
-    constexpr char CalculatorStrings::ERROR_MESSAGE_INPUT_TOO_LARGE[];
-    constexpr char CalculatorStrings::ERROR_MESSAGE_INPUT_TOO_SMALL[];
-    constexpr char CalculatorStrings::RE_ENTER_EQUATION[];
-    constexpr char CalculatorStrings::ENTER_EQUATION[];
-    constexpr char CalculatorStrings::ERROR_MESSAGE_UNKNOWN_OPERATOR[];
-    constexpr char CalculatorStrings::WELCOME_MESSAGE[];
-}
+// ***************************************************************************** /
+// ***************** CalculatorStrings public methods ************************** /
```

Comment:
These aren't methods. ... `static data` might be more appropriate? Or just remove the comment entirely